VERSION.txt

```diff
@@ -1 +1 @@
-28.6.0
+28.6.1
```

Comment:
I'd put this as a minor bump rather than a patch - we're offering new functionality.

Reply:
sounds reasonable 👍